frontend/src/main/pages/CourseDetails/CourseDetailsIndexPage.js

```diff
@@ -26,6 +29,36 @@ export default function CourseDetailsIndexPage() {
     },
   );
 
+  // Parse the subject area and course number, if moreDetails undefined, just leave them blank
+  let subjectArea = "",
+    courseNumber = "";
+  if (moreDetails) {
+    ({ subjectArea, courseNumber } = parseCourseId(moreDetails.courseId));
+  }
+  // Will return grade data or an empty array if moreDetails undefined
+  let {
+    data: gradeData,
+    _error1,
+    _status1,
+  } = useBackend(
+    // Stryker disable all : hard to test for query caching
+    [
+      `/api/gradehistory/search?subjectArea=${subjectArea}&courseNumber=${courseNumber}`,
+    ],
+    {
+      method: "GET",
+      url: `/api/gradehistory/search`,
+      params: {
+        subjectArea,
+        courseNumber,
+      },
+    },
+  );
+
```

Comment:
Need a 
```
// Stryker restore all 
```

Otherwise you are excluding a lot of code from mutation testing that you should not be.

Reply:
Ok that explains why mutation testing told me that most of my tests were useless..